templates/item/item-partials/item-details/details/item-details-attack.hbs

```diff
@@ -1,3 +1,23 @@
 <div>
     <h3>{{localize "ED.Item.General.details"}}</h3>
-</div>
\ No newline at end of file
+</div>
+    {{!-- Target Difficulty --}}
```

Comment:
This should definitely a handlebar partial, as often as it's used.

Reply:
will be refactored